README.md

```
@@ -1,267 +1,216 @@
-    RULE --> ARCHIVE: Graduated (reference only)
-    
-    INSTINCT --> KILLED: Contradicted or idle
-    PATTERN --> INSTINCT: Confidence dropped
-    PATTERN --> KILLED: Contradicted or idle
-    
-    RULE --> META_RULE: 3+ related rules cluster
-    
-    note right of INSTINCT: New lesson, tentative
-    note right of PATTERN: Gaining evidence
-    note right of RULE: Proven behavioral rule
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)